package/batocera/gpu/mali-g31-gbm/mali-g31-gbm.mk

```diff
@@ -7,6 +7,11 @@
 MALI_G31_GBM_VERSION = 6141ad6e6f2d3eb38e7e0962f61b78510b2e2d2c
 MALI_G31_GBM_SITE = $(call github,rockchip-linux,libmali,$(MALI_G31_GBM_VERSION))
 
+# See https://wiki.odroid.com/odroid_go_advance/application_note/vulkan_on_rk3326
+ifeq ($(BR2_PACKAGE_BATOCERA_TARGET_ODROIDGOA),y)
+	MALI_G31_GBM_EXTRA_DOWNLOADS=https://dn.odroid.com/RK3326/ODROID-GO-Advance/rk3326_r13p0_gbm_with_vulkan_and_cl.zip
```

Comment:
nit: No identation within ifs in buildroot code (see e.g. https://github.com/buildroot/buildroot/blob/6e4f081d0c60a5662d401fb87cdd9b66ec397466/package/mesa3d/mesa3d.mk#L32-L34)